zerver/openapi/zulip.yaml

```diff
@@ -4688,6 +4688,13 @@ paths:
                                         guest users to prominently highlight their status.
 
                                         **Changes**: New in Zulip 8.0 (feature level 216).
+                                    enable_guest_user_notification:
+                                      type: boolean
+                                      description: |
+                                        Whether user should be notified when a DM has a guest recipient
+                                        by displaying a compose banner.
```

Comment:
This should be written more clearly about exactly what part of the system should do what.

"Whether clients should display a warning when a user is composing a DM to a guest user in this organization."